collector/stat_progress_vacuum.go

```diff
@@ -49,85 +51,85 @@ func NewStatVacuumProgressScraper() Scraper {
 		running: prometheus.NewDesc(
 			"postgres_stat_vacuum_progress_running",
 			"VACUUM is running",
-			[]string{"pid", "query_start", "datname", "relname"},
+			[]string{"pid", "query_start", "schemaname", "datname", "relname"},
```

Comment:
Need to add the `schemaname` label to all the vacuum metrics rows in the table in the readme.